src/v2/Components/NavBar/NavBar.tsx

```diff
@@ -45,6 +45,22 @@ import Events from "v2/Utils/Events"
 import { useMatchMedia } from "v2/Utils/Hooks/useMatchMedia"
 import { userHasLabFeature } from "v2/Utils/user"
 
+// TODO: Need a place to put global components like this
+const GlobalStyles = createGlobalStyle`
+  /**
+  * This will hide the focus indicator if the element receives focus via the mouse,
+  * but it will still show up on keyboard focus. <input> tags are an exception to this.
```

Comment:
[Inputs are an exception to the keyboard navigation modality](https://github.com/WICG/focus-visible/blob/b29ca5843a48e9fcf5d4407dad1c4458150b5178/src/focus-visible.js#L55-L72)

Reply:
This is actually why I'm using `outline` instead of say for instance, `box-shadow` which would handle border-radius better. Our inputs universally have `outline: 0` and the specificity will win there.